app/javascript/components/Greeting.js

```diff
@@ -7,7 +7,7 @@ const Greeting = () => {
     return (
         greeting.loading ? <h1>Loading...</h1> :
         <div>
-            <h1>{greeting.message}</h1>
+            <h1>{greeting.message}!!</h1>
         </div>
```

Comment:
- [ ] I created some text in the database, but I only saw a loading message throughout.

![loading](https://user-images.githubusercontent.com/53124548/231421785-95d56e00-6d6a-4526-9470-276ae7a63da9.png)